astropy/extern/ply/lex.py

```diff
@@ -119,7 +119,7 @@ def __init__(self):
                                       # regular expression and findex is a list
                                       # mapping regex group numbers to rules
         self.lexretext = None         # Current regular expression strings
-        self.lexstatere = {}          # Dictionary mapping lexer states to master regexs
```

Comment:
leave all extern files as is

Reply:
I must have missed this in my second commit. Good catch!